README.md

```diff
@@ -40,8 +34,6 @@ The following modules are available for use,
 [producers](http://kafka.apache.org/0102/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html)
 and [consumers](http://kafka.apache.org/0102/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html)
```

Comment:
```suggestion
[producers](http://kafka.apache.org/34/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html)
and [consumers](http://kafka.apache.org/34/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html)
```

update version in javadoc link

Reply:
Updated: https://github.com/cerner/common-kafka/pull/51/commits/e758d4484eeb41e10940975cfc8cf4d54b351f73